tests/fixtures/odl.py

```diff
@@ -217,167 +205,3 @@ def opds2_with_odl_api_fixture(
     opds2_with_odl_files_fixture: OPDS2WithODLFilesFixture,
 ) -> OPDS2WithODLApiFixture:
     return OPDS2WithODLApiFixture(db, http_client, opds2_with_odl_files_fixture)
-
-
-class LicenseHelper:
-    """Represents an ODL license."""
-
-    def __init__(
-        self,
-        identifier: str | None = "",
-        checkouts: int | None = None,
-        concurrency: int | None = None,
-        expires: datetime.datetime | str | None = None,
-    ) -> None:
-        """Initialize a new instance of LicenseHelper class.
-
-        :param identifier: License's identifier
-        :param checkouts: Total number of checkouts before a license expires
-        :param concurrency: Number of concurrent checkouts allowed
-        :param expires: Date & time when a license expires
-        """
-        self.identifier = identifier if identifier else f"urn:uuid:{uuid.uuid1()}"
-        self.checkouts = checkouts
-        self.concurrency = concurrency
-        self.expires = (
-            expires.isoformat() if isinstance(expires, datetime.datetime) else expires
-        )
-
-
-class LicenseInfoHelper:
-    """Represents information about the current state of a license stored in the License Info Document."""
-
-    def __init__(
-        self,
-        license: LicenseHelper,
-        available: int,
-        status: str = "available",
-        left: int | None = None,
-    ) -> None:
-        """Initialize a new instance of LicenseInfoHelper class."""
-        self.license: LicenseHelper = license
-        self.status: str = status
-        self.left: int | None = left
-        self.available: int = available
-
-    def __str__(self) -> str:
-        """Return a JSON representation of the License Info Document."""
-        return self.json
-
-    @property
-    def json(self) -> str:
-        """Return a JSON representation of the License Info Document."""
-        return json.dumps(self.dict)
-
-    @property
-    def dict(self) -> dict[str, Any]:
-        """Return a dictionary representation of the License Info Document."""
-        output: dict[str, Any] = {
-            "identifier": self.license.identifier,
-            "status": self.status,
-            "terms": {
-                "concurrency": self.license.concurrency,
-            },
-            "checkouts": {
-                "available": self.available,
-            },
-        }
-        if self.license.expires is not None:
-            output["terms"]["expires"] = self.license.expires
-        if self.left is not None:
-            output["checkouts"]["left"] = self.left
-        return output
-
-
-class OPDS2WithODLImporterFixture:
```

Comment:
Since this fixture is only used in the importer tests, I move it into that file. This doesn't need to be a globally available fixture.